### PR TITLE
Validation implementation for backend

### PR DESCRIPTION
**Pull request by hate-bug**

### CODE REVIEW

**Comment by hate-bug** [comment]
An extra table created here to represent the association between Events and Patients, which also have a boolean state [isValid] to represent if the association is valid or not.

File: pom.xml [hunk 1]
```diff
@@ -42,6 +42,11 @@
             <groupId>com.h2database</groupId>
             <artifactId>h2</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
```
Comment (esfandia) [inline pom.xml:46]: what is the need for this dependency?

File: src/main/java/com/WebController/EventController.java [hunk 2]
```diff
@@ -20,6 +22,9 @@
     @Autowired
     private EventRepository eventRepo;
 
+    @Autowired
+    private AssoicationRepository assoicationRepo;
```
Comment (esfandia) [inline src/main/java/com/WebController/EventController.java:26]: typo in Association (both for class name and var name)

File: pom.xml (hunk 1 above)
Comment (hate-bug) [reply pom.xml:46]: I added that accidentally. I will remove it.

**Pull request merged**